### ISSUE TITLE
Can't build on musl

### ISSUE THREAD

**Issue by motorto** [xmoto/xmoto#145]
Hi, can't seem to build the package for x86_64-musl (voidlinux)

Full log (stdout); https://paste.rs/pEg

CMakeError.log: https://paste.rs/e8F

here is the relevant log:

```
[333/336] Building CXX object src/CMakeFiles/xmoto.dir/xmscene/Zone.cpp.o
[334/336] Building CXX object src/CMakeFiles/xmoto.dir/xmscene/Level.cpp.o
[335/336] Building CXX object src/CMakeFiles/xmoto.dir/xmscene/Scene.cpp.o
[336/336] Linking CXX executable src/xmoto
FAILED: src/xmoto
: && /usr/bin/g++ -DNDEBUG -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2    -fdebug-prefix-map=/builddir/xmoto-0.6.1/build=. -Wl,-z,relro -Wl,-z,now -Wl,--as-needed src/CMakeFiles/xmoto.dir/common/CRCHash.cpp.o src/CMakeFiles/xmoto.dir/common/CameraAnimation.cpp.o src/CMakeFiles/xmoto.dir/common/DBuffer.cpp.o src/CMakeFiles/xmoto.dir/common/Image.cpp.o src/CMakeFiles/xmoto.dir/common/Locales.cpp.o src/CMakeFiles/xmoto.dir/common/Packager.cpp.o src/CMakeFiles/xmoto.dir/common/PolyDraw.cpp.o src/CMakeFiles/xmoto.dir/common/Theme.cpp.o src/CMakeFiles/xmoto.dir/common/VBezier.cpp.o src/CMakeFiles/xmoto.dir/common/VFileIO.cpp.o src/CMakeFiles/xmoto.dir/common/VTexture.cpp.o src/CMakeFiles/xmoto.dir/common/VXml.cpp.o src/CMakeFiles/xmoto.dir/common/WWW.cpp.o src/CMakeFiles/xmoto.dir/common/XMArgs.cpp.o src/CMakeFiles/xmoto.dir/common/XMBuild.cpp.o src/CMakeFiles/xmoto.dir/common/XMSession.cpp.o src/CMakeFiles/xmoto.dir/common/XMSession_default.cpp.o src/CMakeFiles/xmoto.dir/common/svn_version.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_config.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_fixes.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_levels.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_profiles.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_replays.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_srv.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_stats.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_sync.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_themes.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_votes.cpp.o src/CMakeFiles/xmoto.dir/db/xmDatabase_web.cpp.o src/CMakeFiles/xmoto.dir/drawlib/DrawLib.cpp.o src/CMakeFiles/xmoto.dir/drawlib/DrawLibOpenGL.cpp.o src/CMakeFiles/xmoto.dir/gui/basic/GUI.cpp.o src/CMakeFiles/xmoto.dir/gui/basic/GUIButton.cpp.o src/CMakeFiles/xmoto.dir/gui/basic/GUIButtonDrawn.cpp.o src/CMakeFiles/xmoto.dir/gui/basic/GUIConsole.cpp.o src/CMakeFiles/xmoto.dir/gui/basic/GUIEdit.cpp.o src/CMakeFiles/xmoto.dir/gui/basic/GUIFrame.cpp.o src/CMakeFiles/xmoto.dir/gui/basic/GUIList.cpp.o src/CMakeFiles/xmoto.dir/gui/basic/GUIStatic.cpp.o src/CMakeFiles/xmoto.dir/gui/basic/GUITabView.cpp.o src/CMakeFiles/xmoto.dir/gui/specific/GUIXMoto.cpp.o src/CMakeFiles/xmoto.dir/helpers/CmdArgumentParser.cpp.o src/CMakeFiles/xmoto.dir/helpers/Environment.cpp.o src/CMakeFiles/xmoto.dir/helpers/FileCompression.cpp.o src/CMakeFiles/xmoto.dir/helpers/Log.cpp.o src/CMakeFiles/xmoto.dir/helpers/Random.cpp.o src/CMakeFiles/xmoto.dir/helpers/RenderSurface.cpp.o src/CMakeFiles/xmoto.dir/helpers/SwapEndian.cpp.o src/CMakeFiles/xmoto.dir/helpers/System.cpp.o src/CMakeFiles/xmoto.dir/helpers/Text.cpp.o src/CMakeFiles/xmoto.dir/helpers/VMath.cpp.o src/CMakeFiles/xmoto.dir/helpers/utf8.cpp.o src/CMakeFiles/xmoto.dir/image/tim.cpp.o src/CMakeFiles/xmoto.dir/image/tim_io_stdio.cpp.o src/CMakeFiles/xmoto.dir/image/tim_jpeg.cpp.o src/CMakeFiles/xmoto.dir/image/tim_memory_crt.cpp.o src/CMakeFiles/xmoto.dir/image/tim_png.cpp.o src/CMakeFiles/xmoto.dir/net/ActionReader.cpp.o src/CMakeFiles/xmoto.dir/net/NetActions.cpp.o src/CMakeFiles/xmoto.dir/net/NetClient.cpp.o src/CMakeFiles/xmoto.dir/net/NetServer.cpp.o src/CMakeFiles/xmoto.dir/net/ServerRules.cpp.o src/CMakeFiles/xmoto.dir/net/VirtualNetLevelsList.cpp.o src/CMakeFiles/xmoto.dir/net/extSDL_net.cpp.o src/CMakeFiles/xmoto.dir/net/helpers/Net.cpp.o src/CMakeFiles/xmoto.dir/net/thread/ServerThread.cpp.o src/CMakeFiles/xmoto.dir/states/AttractMode.cpp.o src/CMakeFiles/xmoto.dir/states/GameState.cpp.o src/CMakeFiles/xmoto.dir/states/StateCheckWww.cpp.o src/CMakeFiles/xmoto.dir/states/StateCreditsMode.cpp.o src/CMakeFiles/xmoto.dir/states/StateDeadJust.cpp.o src/CMakeFiles/xmoto.dir/states/StateDeadMenu.cpp.o src/CMakeFiles/xmoto.dir/states/StateEditProfile.cpp.o src/CMakeFiles/xmoto.dir/states/StateEditWebConfig.cpp.o src/CMakeFiles/xmoto.dir/states/StateFinished.cpp.o src/CMakeFiles/xmoto.dir/states/StateHelp.cpp.o src/CMakeFiles/xmoto.dir/states/StateLevelInfoViewer.cpp.o src/CMakeFiles/xmoto.dir/states/StateLevelPackViewer.cpp.o src/CMakeFiles/xmoto.dir/states/StateMainMenu.cpp.o src/CMakeFiles/xmoto.dir/states/StateManager.cpp.o src/CMakeFiles/xmoto.dir/states/StateMenu.cpp.o src/CMakeFiles/xmoto.dir/states/StateMessageBox.cpp.o src/CMakeFiles/xmoto.dir/states/StateMultiUpdate.cpp.o src/CMakeFiles/xmoto.dir/states/StateOptions.cpp.o src/CMakeFiles/xmoto.dir/states/StatePause.cpp.o src/CMakeFiles/xmoto.dir/states/StatePlaying.cpp.o src/CMakeFiles/xmoto.dir/states/StatePlayingLocal.cpp.o src/CMakeFiles/xmoto.dir/states/StatePlayingNet.cpp.o src/CMakeFiles/xmoto.dir/states/StatePreplaying.cpp.o src/CMakeFiles/xmoto.dir/states/StatePreplayingCredits.cpp.o src/CMakeFiles/xmoto.dir/states/StatePreplayingGame.cpp.o src/CMakeFiles/xmoto.dir/states/StatePreplayingNet.cpp.o src/CMakeFiles/xmoto.dir/states/StatePreplayingReplay.cpp.o src/CMakeFiles/xmoto.dir/states/StateReplaying.cpp.o src/CMakeFiles/xmoto.dir/states/StateRequestKey.cpp.o src/CMakeFiles/xmoto.dir/states/StateScene.cpp.o src/CMakeFiles/xmoto.dir/states/StateSendReport.cpp.o src/CMakeFiles/xmoto.dir/states/StateSendVote.cpp.o src/CMakeFiles/xmoto.dir/states/StateServerConsole.cpp.o src/CMakeFiles/xmoto.dir/states/StateSync.cpp.o src/CMakeFiles/xmoto.dir/states/StateUpdate.cpp.o src/CMakeFiles/xmoto.dir/states/StateUpdateDb.cpp.o src/CMakeFiles/xmoto.dir/states/StateUpdateRoomsList.cpp.o src/CMakeFiles/xmoto.dir/states/StateUpdateTheme.cpp.o src/CMakeFiles/xmoto.dir/states/StateUpdateThemesList.cpp.o src/CMakeFiles/xmoto.dir/states/StateUpgradeLevels.cpp.o src/CMakeFiles/xmoto.dir/states/StateUploadAllHighscores.cpp.o src/CMakeFiles/xmoto.dir/states/StateUploadHighscore.cpp.o src/CMakeFiles/xmoto.dir/states/StateViewHighscore.cpp.o src/CMakeFiles/xmoto.dir/states/StateVote.cpp.o src/CMakeFiles/xmoto.dir/states/StateWaitServerInstructions.cpp.o src/CMakeFiles/xmoto.dir/states/StateWaiting.cpp.o src/CMakeFiles/xmoto.dir/thread/CheckWwwThread.cpp.o src/CMakeFiles/xmoto.dir/thread/DownloadReplaysThread.cpp.o src/CMakeFiles/xmoto.dir/thread/LevelsPacksCountUpdateThread.cpp.o src/CMakeFiles/xmoto.dir/thread/SendReportThread.cpp.o src/CMakeFiles/xmoto.dir/thread/SendVoteThread.cpp.o src/CMakeFiles/xmoto.dir/thread/SyncThread.cpp.o src/CMakeFiles/xmoto.dir/thread/UpdateDbThread.cpp.o src/CMakeFiles/xmoto.dir/thread/UpdateRoomsListThread.cpp.o src/CMakeFiles/xmoto.dir/thread/UpdateThemeThread.cpp.o src/CMakeFiles/xmoto.dir/thread/UpdateThemesListThread.cpp.o src/CMakeFiles/xmoto.dir/thread/UpgradeLevelsThread.cpp.o src/CMakeFiles/xmoto.dir/thread/UploadAllHighscoresThread.cpp.o src/CMakeFiles/xmoto.dir/thread/UploadHighscoreThread.cpp.o src/CMakeFiles/xmoto.dir/thread/XMThread.cpp.o src/CMakeFiles/xmoto.dir/thread/XMThreadStats.cpp.o src/CMakeFiles/xmoto.dir/thread/XMThreads.cpp.o src/CMakeFiles/xmoto.dir/xmoto/BSP.cpp.o src/CMakeFiles/xmoto.dir/xmoto/Collision.cpp.o src/CMakeFiles/xmoto.dir/xmoto/Credits.cpp.o src/CMakeFiles/xmoto.dir/xmoto/GUIBestTimes.cpp.o src/CMakeFiles/xmoto.dir/xmoto/Game.cpp.o src/CMakeFiles/xmoto.dir/xmoto/GameEvents.cpp.o src/CMakeFiles/xmoto.dir/xmoto/GameInit.cpp.o src/CMakeFiles/xmoto.dir/xmoto/GeomsManager.cpp.o src/CMakeFiles/xmoto.dir/xmoto/Input.cpp.o src/CMakeFiles/xmoto.dir/xmoto/LevelsManager.cpp.o src/CMakeFiles/xmoto.dir/xmoto/LuaLibBase.cpp.o src/CMakeFiles/xmoto.dir/xmoto/LuaLibGame.cpp.o src/CMakeFiles/xmoto.dir/xmoto/Renderer.cpp.o src/CMakeFiles/xmoto.dir/xmoto/RendererFBO.cpp.o src/CMakeFiles/xmoto.dir/xmoto/Replay.cpp.o src/CMakeFiles/xmoto.dir/xmoto/ScriptDynamicObjects.cpp.o src/CMakeFiles/xmoto.dir/xmoto/SomersaultCounter.cpp.o src/CMakeFiles/xmoto.dir/xmoto/Sound.cpp.o src/CMakeFiles/xmoto.dir/xmoto/SysMessage.cpp.o src/CMakeFiles/xmoto.dir/xmoto/Trainer.cpp.o src/CMakeFiles/xmoto.dir/xmoto/Universe.cpp.o src/CMakeFiles/xmoto.dir/xmoto/UserConfig.cpp.o src/CMakeFiles/xmoto.dir/xmoto/VideoRecorder.cpp.o src/CMakeFiles/xmoto.dir/xmoto/VirtualLevelsList.cpp.o src/CMakeFiles/xmoto.dir/xmoto/XMDemo.cpp.o src/CMakeFiles/xmoto.dir/xmoto/XMKey.cpp.o src/CMakeFiles/xmoto.dir/xmscene/BasicSceneStructs.cpp.o src/CMakeFiles/xmoto.dir/xmscene/Bike.cpp.o src/CMakeFiles/xmoto.dir/xmscene/BikeAnchors.cpp.o src/CMakeFiles/xmoto.dir/xmscene/BikeController.cpp.o src/CMakeFiles/xmoto.dir/xmscene/BikeGhost.cpp.o src/CMakeFiles/xmoto.dir/xmscene/BikeParameters.cpp.o src/CMakeFiles/xmoto.dir/xmscene/BikePlayer.cpp.o src/CMakeFiles/xmoto.dir/xmscene/Block.cpp.o src/CMakeFiles/xmoto.dir/xmscene/Camera.cpp.o src/CMakeFiles/xmoto.dir/xmscene/ChipmunkWorld.cpp.o src/CMakeFiles/xmoto.dir/xmscene/Entity.cpp.o src/CMakeFiles/xmoto.dir/xmscene/GhostTrail.cpp.o src/CMakeFiles/xmoto.dir/xmscene/Level.cpp.o src/CMakeFiles/xmoto.dir/xmscene/PhysicsSettings.cpp.o src/CMakeFiles/xmoto.dir/xmscene/Scene.cpp.o src/CMakeFiles/xmoto.dir/xmscene/ScriptTimer.cpp.o src/CMakeFiles/xmoto.dir/xmscene/Serializer.cpp.o src/CMakeFiles/xmoto.dir/xmscene/SkyApparence.cpp.o src/CMakeFiles/xmoto.dir/xmscene/Zone.cpp.o -o src/xmoto  /usr/lib/libbz2.so  src/chipmunk/libchipmunk.a  /usr/lib/libcurl.so  /usr/lib/libjpeg.so  /usr/lib/libxml2.so  src/lua/liblua.a  src/md5sum/libmd5sum.a  /usr/lib/libode.so  src/glad/libglad.a  -ldl  /usr/lib/libOpenGL.so  /usr/lib/libGLX.so  /usr/lib/libGLU.so  /usr/lib/libpng.so  /usr/lib/libSDLmain.a  /usr/lib/libSDL.so  /usr/lib/libSDL_mixer.so  /usr/lib/libSDL_net.so  /usr/lib/libSDL_ttf.so  /usr/lib/libsqlite3.so  /usr/lib/libxdg-basedir.so  /usr/lib/libz.so && :
/usr/bin/ld: src/CMakeFiles/xmoto.dir/common/Locales.cpp.o: warning: relocation against `_nl_msg_cat_cntr' in read-only section `.text'
/usr/bin/ld: src/CMakeFiles/xmoto.dir/common/Locales.cpp.o: in function `Locales::changeLocale(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)':
Locales.cpp:(.text+0x105): undefined reference to `_nl_msg_cat_cntr'
/usr/bin/ld: warning: creating DT_TEXTREL in a PIE
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```

Don't know if I am providing all info.

Thanks for the help

**Comment by Nikekson** [comment]
Should be fixed by 7ce512bf0242182306ceb75d7dbe2370aab838f8

**Comment by motorto** [comment]
Thanks for the quick patch ! 
https://github.com/void-linux/void-packages/pull/39836
